src/Nncase.EGraph/Transform/EGraphPrinter.cs

```diff
@@ -208,7 +208,11 @@ public DotGraph ConvertEGraphAsDot()
     /// <returns>this dot graph.</returns>
     public DotGraph SaveToStream(Stream output)
     {
-        DotGraph.Build(new StreamWriter(output, leaveOpen: true));
+        using (var writer = new StreamWriter(output))
```

Comment:
Should not close the stream here. If users want the stream be leave open, they can't control the behavior of this method. If users want the stream be disposed, they can use `using` at the caller.